test/com/facebook/buck/go/GoTestIntegrationTest.java

```diff
@@ -66,9 +65,7 @@ public void testGoTest() throws IOException {
   @Test
   public void testGoTestTimeout() throws IOException {
     ProjectWorkspace.ProcessResult result = workspace.runBuckCommand("test", "//:test-spinning");
-    result.assertSpecialExitCode("test should fail", 42);
-    String stderr = result.getStderr();
-    assertTrue(stderr, stderr.contains("Timed out running test command"));
+    result.assertTestFailure("test timed out");
```

Comment:
@dreiss just pushed 7c2311282079c9e06117974b1d025b0079452d19 which conflicts with this, and I want to make sure that the end result should still be the same here.